integration_tests/CMakeLists.txt

```diff
@@ -816,6 +816,7 @@ RUN(NAME callback_04       IMPORT_PATH .. LABELS cpython)
 # Intrinsic Functions
 RUN(NAME intrinsics_01     LABELS cpython llvm NOFAST) # any
 RUN(NAME intrinsics_02     LABELS cpython llvm c) # floordiv
+RUN(NAME intrinsics_03     LABELS llvm) # lshift
```

Comment:
```suggestion
RUN(NAME intrinsics_03     LABELS cpython llvm) # lshift
```

This verifies that the test works with python as well.

Reply:
Please check the comment below: https://github.com/lcompilers/lpython/pull/2556#issuecomment-1983179803